INN/umbrella-sfpublicpress
# Final qa

## Changes

This pull request makes the following changes:

- [x] Header styles should be consistent across site and set to:
16, 28, 48, 54 is biggest
- [x] Small items 2rem
- [x] Bigger 3.25rem

**Homepage**
- [x] Remove extra stuff in bylines (job titles) from homepage views
- [x] Remove bylines altogether from right gray box
- [x] Global nav & top bar:
    - [x] Black bar height 39
    - [x] Font size 16
- [x] Font size 16 for bylines & category tags on homepage
- [x] Main headline (H2):
    - [x] font size 54px
    - [x] Line 3.25
- [x] Other headlines on home (H5):
    - [x] Line height 3.25 rem
- [x]  Gray box headlines:
    - [x] 2 rem
    - [x] 26px
- [x] Full story link: 16px .view.more.link
- [x] Section titles: 54px
- [x] Podcast titles: 2 rem widget.sppublicpress-podcasts H5

![screencapture-sfpp-test-2020-06-11-13_02_40](https://user-images.githubusercontent.com/18353636/84417720-10256200-abe4-11ea-8aaf-acac9c4ddb6d.png)

**Author/interior pages**:
- [x] Page titles: 
    - [x] 70px, 5rem
- [x] Get author image to left of text, or something, pull circle styling from articles
- [x] Remove sidebar
- [x] Text black, left align
- [x] Page title to left

![screencapture-sfpp-test-tag-coronavirus-2020-06-11-13_04_02](https://user-images.githubusercontent.com/18353636/84417742-17e50680-abe4-11ea-9e60-b1fc4232e67c.png)

![screencapture-sfpp-test-author-brian-howey-2020-06-11-13_03_50](https://user-images.githubusercontent.com/18353636/84417759-1c112400-abe4-11ea-80bf-1450769f73e6.png)

**Article pages**:
- [x] Headline: 
       - [x] 70px
       - [x] Line: 5rem
- [x] Featured images shorter - how to approach?
- [x] Body text 20px
- [x] Related stories:
        - [x] Match sizes elsewhere for widget title
        - [x] Same with headlines (match grey box styles)

![screencapture-sfpp-test-oakland-lifts-city-curfew-after-thousands-defied-it-wednesday-2020-06-11-13_03_22](https://user-images.githubusercontent.com/18353636/84417812-2fbc8a80-abe4-11ea-9ec7-18b391158e06.png)

## Why



For #86

## Testing/Questions

Features that this PR affects:

- Mostly all styles on the site


Questions that need to be answered before merging:


Steps to test this PR:

1. Poke around the homepage and make sure nothing is broken
2. Poke around a category/tag archive and make sure nothing is broken
3. Poke around an author archive and make sure nothing is broken
4. Poke around an article and make sure nothing is broken
